venues/ICLR.cc/2019/Conference/python/iclr19.py

```diff
@@ -443,6 +443,21 @@
         ],
         'invitees': [],
         'web': os.path.abspath('../webfield/bidWebfield.js')
+    },
+    content_params = {
```

Comment:
if this is going to be our general solution for other conferences, this content should be changed in openreview-py

Reply:
see here: https://github.com/iesl/openreview-py/pull/163